Fail to build current oiio-git...

Enviroment:
Ubuntu 16.04.1 + GCC/G++ 5.4.0


> [100%] Generating idiff.1
> cd ......//oiio/build/src/doc && ../idiff/idiff --help | /usr/bin/python ......//oiio/src/doc/help2man_preformat.py | /usr/bin/txt2man -v OpenImageIO -s 1 -t idiff > ......//oiio/build/src/doc/idiff.1
> 
> 
> make[2]: Leaving directory '......//oiio/build'
> CMakeFiles/Makefile2:1906: recipe for target 'src/doc/CMakeFiles/man_pages.dir/all' failed
> make[1]: *** [src/doc/CMakeFiles/man_pages.dir/all] Error 2
> make[1]: Leaving directory '......//oiio/build'
> Makefile:152: recipe for target 'all' failed
> make: *** [all] Error 2
> jiapei:build$ pwd
> ......//oiio/build
> jiapei:build$ ls
> 



is oiio compatible with python3 now?


Cheers
Pei

A bit improvement, but still errors:




> make -f src/doc/CMakeFiles/man_pages.dir/build.make src/doc/CMakeFiles/man_pages.dir/depend
> make[2]: Entering directory '......//oiio/build'
> cd ......//oiio/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" ......//oiio ......//oiio/src/doc ......//oiio/build ......//oiio/build/src/doc ......//oiio/build/src/doc/CMakeFiles/man_pages.dir/DependInfo.cmake --color=
> make[2]: Leaving directory '......//oiio/build'
> make -f src/doc/CMakeFiles/man_pages.dir/build.make src/doc/CMakeFiles/man_pages.dir/build
> make[2]: Entering directory '......//oiio/build'
> make[2]: *** No rule to make target '../src/doc/iv', needed by 'src/doc/iv.1'.  Stop.
> make[2]: Leaving directory '......//oiio/build'
> CMakeFiles/Makefile2:2802: recipe for target 'src/doc/CMakeFiles/man_pages.dir/all' failed
> make[1]: *** [src/doc/CMakeFiles/man_pages.dir/all] Error 2
> make[1]: Leaving directory '......//oiio/build'
> Makefile:163: recipe for target 'all' failed
> make: *** [all] Error 2
> jiapei:build$ ccmake ../
>

@lgritz 

> jiapei:build$ make nuke
> make: *** No rule to make target 'nuke'.  Stop.
> jiapei:build$ ls
> build.log  CMakeCache.txt  CMakeFiles  cmake_install.cmake  cpack  CPackConfig.cmake  CPackSourceConfig.cmake  CTestTestfile.cmake  DartConfiguration.tcl  include  Makefile  src  Testing  testsuite
> jiapei:build$ 


Cheers
Pei

Current Build Error Message:

> [ 99%] Generating iinfo.1
> cd ....../oiio/build/src/doc && ../oiiotool/oiiotool --help | /usr/bin/python3.5 ....../oiio/src/doc/help2man_preformat.py | /usr/bin/txt2man -v OpenImageIO -s 1 -t oiiotool > ....../oiio/build/src/doc/oiiotool.1
> cd ....../oiio/build/src/doc && ../iinfo/iinfo --help | /usr/bin/python3.5 ....../oiio/src/doc/help2man_preformat.py | /usr/bin/txt2man -v OpenImageIO -s 1 -t iinfo > ....../oiio/build/src/doc/iinfo.1
> 
> 
> make[2]: Leaving directory '....../oiio/build'
> CMakeFiles/Makefile2:2882: recipe for target 'src/doc/CMakeFiles/man_pages.dir/all' failed
> make[1]: *** [src/doc/CMakeFiles/man_pages.dir/all] Error 2
> make[1]: Leaving directory '....../oiio/build'
> Makefile:152: recipe for target 'all' failed
> make: *** [all] Error 2
> 


Cheers
Pei

--- a/src/doc/CMakeLists.txt
+++ b/src/doc/CMakeLists.txt
@@ -18,7 +18,7 @@ find_package(PythonInterp)
 if (UNIX AND TXT2MAN AND PYTHONINTERP_FOUND)
     message (STATUS "Unix man page documentation will be generated")
     set (cli_tools oiiotool iinfo maketx idiff igrep iconvert)
-    if (USE_QT)
+    if (IV_BUILT)
         list (APPEND cli_tools iv)
     endif()
 

--- a/src/iv/CMakeLists.txt
+++ b/src/iv/CMakeLists.txt
@@ -24,6 +24,7 @@ if (QT4_FOUND AND OPENGL_FOUND AND GLEW_FOUND)
                                ${Boost_LIBRARIES} ${CMAKE_DL_LIBS}
                                ${GLEW_LIBRARIES})
     oiio_install_targets (iv)
+    set (IV_BUILT 1)
 
 else ()
     message (STATUS "\n\n   WARNING: Qt, OpenGL, or GLEW not found -- 'iv' will not be built!\n")

